### PR TITLE
Add unformat of currency

### PR DESCRIPTION
**Pull request by stenehall**

### CODE REVIEW

**Comment by stenehall** [comment]
New fancy unformat of currency, have a look at the tests and see if you agree @iZettle/web

File: src/utils/locale.test.js [hunk 1]
```diff
@@ -90,6 +90,19 @@ describe("locale", () => {
...
   })
 
+  describe("toUnformattedNumber()", () => {
+    it("should unformat any type of currency to smallest currency point", () => {
+      const locale = createLocale()
+      expect(locale.toUnformattedNumber("10")).toEqual(1000)
+      expect(locale.toUnformattedNumber("10,00")).toEqual(1000)
+
+      expect(locale.toUnformattedNumber("10.00")).toEqual(100000)
+      expect(locale.toUnformattedNumber("10.10,00")).toEqual(101000)
```
Comment (adamvoncorswant) [inline src/utils/locale.test.js]: This is highly locale dependent. Do we want to ensure that we can handle different locales, such as:
10.100,00
10,100.00
?
Also, should we have tests with a currency with zero (JPY) or possibly three decimal denominators?

Comment (stenehall) [inline src/utils/locale.test.js]: These tests are done using a bootstraped example locale. In the example locale we're using , as a "öre" delimiter and 2 decimals. It's showing that all of those works. We could for sure add another example locale, but the tests do show that it's all working.

Comment (adamvoncorswant) [inline src/utils/locale.test.js]: Well, to me that looks like selective ignorance :)
As this is a public repository it would be nice to put the burden of proof on the test cases...

Anyway: do as you wish!

Comment (stenehall) [inline src/utils/locale.test.js]: No worries, I do agree a bit with you. I'm adding more tests. However the major of what we'd be testing with more tests would be testing accounting.js and that should test itself.

Comment (stenehall) [inline src/utils/locale.test.js]: And as it turns out it was good to do, some edge cases that we should decide on.

Comment (adamvoncorswant) [inline src/utils/locale.test.js]: Great stuff! Thanks for listening :D :cake:

Comment (stenehall) [inline src/utils/locale.test.js]: There we go, 3 different types of decimal delimiters as well as thousands separator and 2 different decimal points. I'm not sure what to do with zero decimal points in Yen when you actually get decimals, atm I'm rounding it. Are we ok with that?

Comment (adamvoncorswant) [inline src/utils/locale.test.js]: Great! Yes, sounds like a good approach for the purpose of this library!
